components/vote/button/button.jsx

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 require("./button-style.scss");
 
 export default (props) => {
-  let { value, myValue, maxDown, maxUp, color, onVote, className } = props;
+  let { value, myValue, maxDown, maxUp, color, onVote, className, isLoginActive } = props;
+  let {voteAppToken} = localStorage;
```

Comment:
We should not let this component know about the login system (`localStorage`, `voteAppToken`). Could this information just be passed via props i. e `loggedIn = true` or `myValue != null`?